packages/vm/tests/instanceof.test.ts

```diff
@@ -4,6 +4,30 @@
 
 import { EdgeVM } from '../src'
 
+it('DOMException', async () => {
```

Comment:
question: Is this test related to this upgrade?

Reply:
It's a nice-to-have test. The Node.js `DOMException` implementation is not extending from `Error`. I wanted to verify `new DOMException() instanceof Error === true` before and after shipping this.

Reply:
The Node.js DOMException does extend Error:

```sh-session
⛧ node -p 'new DOMException() instanceof Error'
true
```

how did you fix it? You DM'd me that it doesn't work <picture data-single-emoji=":lol:" title=":lol:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol/5986e156eca2154f.png" alt=":lol:" align="absmiddle"></picture>

Reply:
btw from which version of Node.js DOMException exists? I just checked the following

```sh-session
⛧ fnm use 16
Using Node v16.18.1
⛧ node -p 'new DOMException() instanceof Error'
[eval]:1
new DOMException() instanceof Error
^

ReferenceError: DOMException is not defined
```

Reply:
The `DOMException` built-in in Node.js is not exactly extending from `Error`:
https://github.com/nodejs/node/blob/62b2cf30f2d1326dde9d4bc047f5611f17c4a20f/lib/internal/per_context/domexception.js#L53

That isn't a problem, unless you use a VM <picture data-single-emoji=":sadblob:" title=":sadblob:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/sadblob/240fdcad6c901eef.gif" alt=":sadblob:" align="absmiddle"></picture> 

so what I did is to just keep our polyfill (which is pretty minimal) and in that way the inheritance is correct.